packages/block-library/src/columns/edit.native.js

```diff
@@ -44,6 +44,13 @@ const ALLOWED_BLOCKS = [ 'core/column' ];
 const DEFAULT_COLUMNS = 2;
 const MIN_COLUMNS_NUMBER = 1;
 
+/**
+ * Number of columns in each row for large breakpoint container width
+ *
+ * @type {number}
+ */
+const LARGE_CONTAINER_COLUMNS_IN_ROW = 3;
```

Comment:
Wdyt about renaming `LARGE_CONTAINER_COLUMNS_IN_ROW` to `MAX_COLUMNS_NUM_IN_ROW`? Then we can also rename:

`DEFAULT_COLUMNS` to `DEFAULT_COLUMNS_NUM` and `MIN_COLUMNS_NUMBER` to `MIN_COLUMNS_NUM` to keep the consistency ✌️

Reply:
Hello @lucasstark 👋 
Your proposal sounds good. I submitted the changes.

Reply:
Thanks!